lib/LaTeXML/Package/TeX.pool.ltxml

```diff
@@ -4229,15 +4229,9 @@ DefMathLigature("!!" => "!!", role => 'POSTFIX', meaning => 'double-factorial');
 # Recognize :=
 DefMathLigature(":=" => ":=", role => 'RELOP', meaning => 'assign');
 
-# Recognize: <=, << and <<<
+# Recognize: <<< and >>>
 # # Note that in <<<, the leading << will already have been converted to \ll !!!
-DefMathLigature("<="        => "\x{2264}", role => 'RELOP', meaning => 'less-than-or-equals');
-DefMathLigature("<<"        => "\x{226A}", role => 'RELOP', meaning => 'much-less-than');
 DefMathLigature("\x{226A}<" => "\x{22D8}", role => 'RELOP', meaning => 'very-much-less-than');
```

Comment:
Ugh. is this even a thing? Maybe it should just be removed, since it most likely can't even fire now? is it worth handling in the grammar at all?

Reply:
Indeed, this ligature will never match now. A quick grep through the arXiv data shows 50-100  of `very-much-less-than` matches, but they all seem connected to an explicit `\lll` macro, and one could argue that the `>>>` in math mode TeX looks really unpleasant in the first place... Maybe I just remove the ligature entirely?